lib/coney/consumer_executor.ex

```diff
@@ -8,23 +8,6 @@ defmodule Coney.ConsumerExecutor do
     |> consumer.parse(meta)
     |> consumer.process(meta)
     |> handle_result(task)
-  rescue
```

Comment:
These are still good i believe, if something raises, it will catch the raise at least.